src/core/qgsmbtilesreader.cpp

```diff
@@ -46,6 +49,37 @@ bool QgsMBTilesReader::isOpen() const
   return bool( mDatabase );
 }
 
+bool QgsMBTilesReader::create()
```

Comment:
I missed this on earlier reviews, but "MB" should be "Mb" for consistency with the rest of the API